{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/templates/registration/password_reset_code.html

```diff
@@ -0,0 +1,41 @@
+{%- raw -%}
+{% extends '_action-email-base.html' %}
+{% load tz %}
```

Comment:
@oudeismetis not sure what to do about this file though

Reply:
files are easy actually.
Add logic to the post_gen_project hook to delete the file if users didn't pick the option.

Reply:
Addressed!
https://github.com/thinknimble/tn-spa-bootstrapper/pull/192/commits/09d29a291604def39db4d7ec4bb1c12d0156d615